.github/workflows/test.yaml

```diff
@@ -19,8 +19,7 @@ jobs:
         pip install flake8 black
     - name: Test code quality
       run: |
-        flake8 --count --show-source --statistics
-        black -v --check dodola/*
+        make format-check
```

Comment:
@brews this is just to make the syntax easier, right ? 

I.e instead of typing `flake8 --count --show-source --statistics` I'll type `make format-check`.

Reply:
Yeah, that's the idea. I've found myself looking into `.github/workflows/test.yaml` to figure out the exact flags we used for flake8, and black, etc. That's usually a sign to think about simplifying things.

It might make it easier to write a CONTRIBUTING file and other documentation too.

Reply:
@brews oh yes I did that many times... that's useful thanks !